Frontend/src/views/MapView.vue

```diff
@@ -1,15 +1,18 @@
 <template>
   <div>
-    <div v-on:click="requestPoints" id="map"></div>
+   <div><PointDetails></PointDetails></div>
+          <div v-on:click="requestPoints" id="map">
+          </div> 
   </div>
 </template>
 
 <script>
 import {getPoints} from '../services/pointServices'
 import {gmapApi} from 'vue2-google-maps'
 import {checkSession} from '../services/authorizationService'
-
+import {PointDetails} from '../views/PointDetails'
 export default {
+  components: {PointDetails},

```

Comment:
what is this line doing?

Reply:
I was trying to get both components on the same page, but decided just to make it a separate page for simplicity sake